build_helper.sh

```diff
@@ -58,9 +58,6 @@ function prepare_upload_data {
   rm -rf ${LOCAL_PATH}
   mkdir -p ${LOCAL_PATH}
   mv handler.zip ${LOCAL_PATH}/lambda.zip
-}
-
-function prepare_cloudformation_code {
```

Comment:
I introduced this function while refactoring the Makefile a few weeks back, but it was never called and now it doesn't make sense anymore.